commerce_coordinator/settings/local.py

```diff
@@ -127,9 +128,9 @@
 
 PAYMENT_MICROFRONTEND_URL = "http://localhost:1998"
 
-TITAN_URL = "http://example.com"
+TITAN_URL = "https://theseus.staging.getsmarter.com/"
 
-TITAN_OAUTH2_PROVIDER_URL = "http://example.com"
+TITAN_OAUTH2_PROVIDER_URL = "https://theseus.staging.getsmarter.com/"
```

Comment:
I think we should not push this test code.

Reply:
removed